pkg/router/template/router.go

```diff
@@ -428,9 +464,19 @@ func (r *templateRouter) writeDefaultCert() error {
 // watchMutualTLSCert watches the directory containing the certificates for
 // mutual TLS and reloads the router if the directory contents change.
 func (r *templateRouter) watchMutualTLSCert() error {
+	var caUpdateChannel chan struct{}
 	caPath := os.Getenv("ROUTER_MUTUAL_TLS_AUTH_CA")
+	// TODO: Determine the best place to put CRLs
+	crlPath := "/crls/crls.pem"
+	os.Setenv("ROUTER_MUTUAL_TLS_AUTH_CRL", crlPath)
```

Comment:
This looks a bit hacky.  Could you add a new field to `templateData` and use that instead?

Reply:
Agreed, this is definitely hacky. With this change the actual CRL path is going to be constant, so I can probably change the template to include a `crl-file /crls/crls.pem` (or wherever it's decided the CRL file will live) whenever `ROUTER_MUTUAL_TLS_AUTH_CA` is set. What do you think of that approach?